### PR TITLE
Add support for Customer business_vat_id

### PR DESCRIPTION
**Pull request by guillaumehenriot**
Added support for the business_vat_id property in customers: https://stripe.com/docs/api#create_customer-business_vat_id

### CODE REVIEW

**Comment by guillaumehenriot** [comment]
r? @brandur 
forgot that line...

File: customer.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ type CustomerParams struct {
 	TrialEnd      int64
 	DefaultSource string
 	Shipping      *CustomerShippingDetails
+	BusinessVatId string
```
Comment (brandur) [inline customer.go]: Would you mind renaming this to `BusinessVatID` just to be more in-line with other identifiers in the project? [For example](https://github.com/stripe/stripe-go/blob/adffbfe3d540c9cb8fd8d7962df88b6c86714583/discount/client.go#L31).

It maybe should be `BusinessVATID`, but that looks like we're already using `BusinessVatID` elsewhere in the project.

**Comment by brandur** [comment]
Left a minor comment above, but looks good. Thanks!

**Comment by guillaumehenriot** [comment]
Good catch! I changed it.

**Comment by brandur** [comment]
@guillaumehenriot Thanks! But actually, would you mind going `BusinessVatID`? Even if it's less strictly correct, I noticed [that this capitalization is already in use elsewhere in the project](https://github.com/stripe/stripe-go/blob/adffbfe3d540c9cb8fd8d7962df88b6c86714583/account.go#L212), so we should stick to it.

**Comment by guillaumehenriot** [comment]
@brandur I'm really sorry, I read your comment too quickly and went for the wrong version. Changed it to the good one now!

**Comment by brandur** [comment]
Awesome!

LGTM.

**Comment by brandur** [comment]
Released as 15.1.0.

**Pull request merged**